aequilibrae/paths/traffic_assignment.py

```diff
@@ -474,12 +480,89 @@ def info(self) -> dict:
                 'Target RGap': self.assignment.rgap_target}
         return info
 
-    def save_skims(self, matrix_name: str, which_ones='final') -> None:
+    def save_skims(self, name: str, which_ones='final', format='omx') -> None:
         """Saves the skims (if any) to the skim folder and registers in the matrix list
 
         Args:
-            matrix_name (:obj:`str`): Name of the file to hold this matrix
+            name (:obj:`str`): Name of the file to hold this matrix
```

Comment:
i'd call this filename

Reply:
fair

Reply:
oh, except it's not until you add an "omx" / "aem" extension...